lark/common.py

```diff
@@ -24,6 +37,7 @@ class LexerConf(Serialize):
     g_regex_flags: int
     skip_validation: bool
     use_bytes: bool
+    lexer_type: Optional[_LexerArgType]
```

Comment:
Wasn't initially annotated and the initial assignment to `None` made `mypy` think the value could only be `None`.